packages/ppbgdi/data_stream/cloudfront_logs/fields/fields.yml

```diff
@@ -32,3 +32,55 @@
       type: long
       description: |-
         When the response contains the HTTP Content-Range header, this field contains the range end value.
+    - name: cache_behavior.path_pattern
+      type: keyword
+      description: |-
+        The path pattern that identifies the cache behavior that matched the viewer request.
+    - name: client.country
+      type: keyword
+      description: |-
+        A country code that represents the viewer's geographic location, as determined by the viewer's IP address.
+        For a list of country codes, see https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
+    - name: cs_header_names
+      type: keyword
+      description: |-
+        The names of the HTTP headers (not values) in the viewer request. Note: This field is truncated to 800 bytes.
+    - name: cs_headers_count
+      type: integer
+      description: |-
+        The number of HTTP headers in the viewer request.
+- name: aws.firehose
+  type: group
+  fields:
+    - name: arn
+      type: keyword
+      description: |
+        Firehose ARN.
+    - name: request_id
+      type: keyword
+      description: |
+        Firehose request ID.
+    - name: subscription_filters
+      type: keyword
+      description: |
+        Firehose request ID.
+    - name: parameters.*
+      type: flattened
+      description: |
+        Key-value pairs users set up when creating the Kinesis Data Firehose. These parameters are included in each HTTP call.
+- name: aws.kinesis
+  type: group
+  fields:
+    - name: name
+      type: keyword
+      description: |
+        Kinesis name.
+    - name: type
+      type: keyword
+      description: |-
+        Kinesis type.
+- name: http.request.headers.*
+  type: keyword
+  description: Not yet but already in use ecs fields
```

Comment:
Maybe it's too late but I don't really get what you wanna say with this description :smile: